src/cluster/__tests__/committedOffsets.spec.js

```diff
@@ -0,0 +1,46 @@
+const { createCluster } = require('testHelpers')
+
+describe('Cluster', () => {
+  let groupId
+
+  beforeEach(() => {
+    groupId = 'test-group-id'
+  })
+
+  const testWithOffsets = offsets => {
```

Comment:
Since we only have 2 tests, let's split them into separate cases instead of using the function to generate the tests